cirq/contrib/quirk/url_to_circuit.py

```diff
@@ -46,10 +46,12 @@ def quirk_url_to_circuit(
             qubits). The maximum number of qubits in a Quirk circuit is 16.
             This argument defaults to `cirq.LineQubit.range(16)` when not
             specified.
-        extra_cell_makers: A list of non-standard Quirk cell makers. This can be
+        extra_cell_makers: Non-standard Quirk cells to accept. This can be
             used to parse URLs that come from a modified version of Quirk that
-            includes gates that Quirk doesn't define. See
-            `cirq.contrib.quirk.cells.CellMaker`.
+            includes gates that Quirk doesn't define. This can be specified
+             as either a list of `cirq.contrib.quirk.cells.CellMaker` instances,
```

Comment:
nit: this and two following lines are indented by one more space than they should be